Github-Org-Management-Policy.md

```diff
@@ -0,0 +1,50 @@
+# Node.js GitHub Organization Management Policy
+
+The Node.js Foundation Github Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the
+operational direction of the Node.js Technical Steering Committee (TSC).
+
+## Organization Roles
+
+Individual users within a GitHub organization may be granted one of three
+distinct roles: Member, Owner, or Billing manager. The permissions granted
+to each role are documented in the GitHub documentated [Permission levels for
+an organization](https://help.github.com/articles/permission-levels-for-an-organization/).
+
+### Billing manager
+
+All Billing managers for the Node.js Foundation GitHub Organization shall
+be determined by the Node.js Foundation Board of Directors with notification
+given to the TSC.
+
+### Owners
+
+All voting members of the Node.js TSC shall have Owner permissions within the
+Node.js Foundation GitHub Organization. Additional users may be granted Owner
+permissions at the discretion of the TSC through normal TSC motion and simple
+majority vote.
```

Comment:
kk. Down the road, I think it would be good to sort of create our own roles that correspond to access levels within the bot, and then start getting really stringent on who we give ownership rights too (like, not even TSC by default)...but we're not ready for that yet, so it's probably premature to bake that language in now, given that we want to do both.